### PR TITLE
Home Page design

### PR DESCRIPTION
**Pull request by Deependra3010**

### CODE REVIEW

File: client/src/components/Record.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+import { IExpense } from '../../../server/models/Expense';
```
Comment (Deependra3010) [inline client/src/components/Record.tsx]: '@server/models/Expense'.

File: client/src/components/HomeScreenRecords.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    color: #222;
+    font-size: 18px;
+`;
+const LightText = styled(Link)`
+    color: #666;
+    font-size: 14px;
+`;
+
+const HomeScreenRecords: React.FC = () => {
+    const [records, setRecords] = useState<any[]>([]);
```
Comment (Deependra3010) [inline client/src/components/HomeScreenRecords.tsx]: Todo: <any[]> :eyes:

File: client/src/pages/HomePage.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        margin-top: 0;
+    }
+`;
+const BalanceCard = styled.div`
+    border-radius: 20px;
+    background: #2f7e79;
+    padding: 20px;
+    width: 80%;
+    margin-left: auto;
+    margin-right: auto;
```
Comment (Adityamaurya123uma) [inline client/src/pages/HomePage.tsx]: This can be done in one line, ```margin: 0 auto;```

File: client/src/pages/HomePage.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    &:hover {
+        box-shadow: 0px 0px 5px #fff;
+    }
+`;
+const IncomeExpense = styled.span`
+    color: #d0e5e4;
+    font-family: Inter;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 500;
```
Comment (Adityamaurya123uma) [inline client/src/pages/HomePage.tsx]: You can use this,
```
color: #d0e5e4;
font: 500 16px Inter;
```
Not a big issue using your version but, it looks cleaner.

File: client/src/pages/Login.tsx [hunk 5]
```diff
@@ -16,24 +15,26 @@ const LoginImage = styled.div`
...
     font-weight: 700;
     margin-top: 30px;
     width: 100%;
 `;
-const LoginBtn = styled.button`
+const LoginBtn = styled(Link)`
     border: none;
     color: #fff;
     font-size: 18px;
     font-weight: 600;
```
Comment (Adityamaurya123uma) [inline client/src/pages/Login.tsx:29]: Same here.

File: client/src/pages/Login.tsx [hunk 6]
```diff
@@ -16,24 +15,26 @@ const LoginImage = styled.div`
     }
 `;
 const LoginText = styled.div`
-    color: ${colors.green};
+    color: var(--green);
     text-align: center;
     font-size: 36px;
     font-weight: 700;
```
Comment (Adityamaurya123uma) [inline client/src/pages/Login.tsx:21]: Same here.

**Review by Adityamaurya123uma — APPROVED** [review]

**Review by Adityamaurya123uma — CHANGES_REQUESTED** [review]
Looks good. Just few small changes.

File: client/src/pages/Login.tsx (hunk 5 above)
Comment (Deependra3010) [reply client/src/pages/Login.tsx:29]: Here it won't work as `font` shorthand property requires at least the font-size and font-family values to be specified. here we dont have `font-family`.

File: client/src/pages/Login.tsx (hunk 6 above)
Comment (Deependra3010) [reply client/src/pages/Login.tsx:21]: Same here we dont have `font-family` so short hand wont work.

**Comment by Deependra3010** [comment]
@Adityamaurya123uma How about now!!!

**Review by Adityamaurya123uma — APPROVED** [review]

**Comment by Adityamaurya123uma** [comment]
LGTM

**Pull request merged**